.vscode/launch.json

```diff
@@ -50,7 +50,8 @@
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurevirtualmachines",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azureappservice",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-cosmosdb",
-                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurecontainerapps"
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurecontainerapps",
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurespringcloud"
```

Comment:
If this directory (or any of these) don't exist, does VSCode get upset, or just print an error somewhere and continue?

Reply:
Did a quick test, and I don't have any warnings or errors come up if I try to debug with `"--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurespringcloud"` in the `launch.json`. I checked the output, the debug console, and the developer tools to see if there was any error thrown. 

Seems oaky.